src/material/core/_core.scss

```diff
@@ -16,20 +22,11 @@
   @include text-field.text-field-autosize();
   @include text-field.text-field-autofill();
 
-  @include focus-indicators.private-strong-focus-indicators-positioning();
-  @include _mdc-core();
-}
-
-// Mixin that renders all of the core MDC styles. Private mixin included with `mat-core`.
-@mixin _mdc-core() {
-  @include _mdc-strong-focus-indicators-positioning();
-}
+  .mat-focus-indicator {
```

Comment:
If you want, I think you could move the `.mat-focus-indicator { ... }` wrapper inside the structural styling mixin, and just do:

```
@include focus-indicators.private-structural-styling('mat');
```

here. Then you could determine the proper selector within the mixin using the `$prefix` param. And same below with the MDC mixin include as well.